### PR TITLE
ooniprobe 1.5.1

### PR DESCRIPTION
**Pull request by hellais**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---
- Also update the sub-dependencies of it

### CODE REVIEW

File: Formula/ooniprobe.rb [hunk 1]
```diff
@@ -169,10 +167,6 @@ def install
     (var/"lib/ooni").mkpath
   end
 
-  def post_install
-    system bin/"ooniresources", "--update-inputs", "--update-geoip"
```
Comment (DomT4) [inline Formula/ooniprobe.rb]: No longer needed?

Comment (hellais) [inline Formula/ooniprobe.rb]: Yes, it's now run as part of `setup.py` (https://github.com/TheTorProject/ooni-probe/blob/master/setup.py#L166)

Comment (DomT4) [inline Formula/ooniprobe.rb]: I'm not sure in our case, given we bottle things, that doesn't mean we don't want to retain this. Most people will pour the bottles, and for example the bottles are generated today so someone installing in a couple of months would still get the `inputs` and `geoip` from today as-is.

Comment (hellais) [inline Formula/ooniprobe.rb]: Oh I see. I didn't think about the bottles.

Will add it back then and take note of the fact that in the future ooniprobe should support disabling the fetching of resources via setup.py for use-cases like this.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/bf71e3cb77535d08f7596865449339b758b8ce4b. Thank you for your contribution to Homebrew @hellais; we appreciate it! :smiley_cat:

**Pull request closed without merging**